### ISSUE TITLE
Sometimes, right-clicking in the disassembly view resumes emulation

### ISSUE THREAD

**Issue by adriweb** [CE-Programming/CEmu#503]
From @runer112 on Discord:

> **Runer112** — _Today at 18:41_
> I'm not sure exactly what's happening, but sometimes, right-clicking in the disassembly view resumes emulation
> makes it kinda hard to set breakpoints

**Comment by mateoconlechuga** [comment]
Probably clicking `Run Until`

**Comment by adriweb** [comment]
Yeah, what Mateo said, probably, since at the time this was reported, it was the first context-menu entry, and selecting it is the only path from that menu that resumes emulation.

Anyway `Run Until` isn't the first entry anymore, so this may already be incidentally "resolved" lol
<img width="510" height="206" alt="Image" src="https://github.com/user-attachments/assets/73cc21cd-23cb-402a-83eb-1077e52e7810" />

(Also, here on macOS the menu item is quite a few pixels away so an accidental click (or release I suppose) isn't possible in the first place)
